src/components/tables/adoptionReportTable/index.tsx

```diff
@@ -32,6 +33,7 @@ const AdoptionReportTable: React.FC<AdoptionReportTableProps> = ({
         title={"Adopter's Email"}
         dataIndex={'email'}
         key={'email'}
+        render={(email: string) => <CopyText text={email} />}
```

Comment:
Weird that theres no AntD copy text component, but if it works!

Is there a visual indication that you copied something? ❓